examples/ui-demo/package.json

```diff
@@ -23,7 +23,8 @@
     "react-dom": "^18",
     "tailwind-merge": "^2.3.0",
     "tailwindcss-animate": "^1.0.7",
-    "viem": "^2.9.28"
+    "viem": "^2.9.28",
+    "wagmi": "^2.10.7"
```

Comment:
we should probs update the rest of the SDK to be on these versions. The SDK is currently expecting you to be pinned at `2.8.6` (https://github.com/alchemyplatform/aa-sdk/blob/v4.x.x/aa-sdk/core/package.json#L59)